apisix/stream/xrpc/protocols/redis/init.lua

```diff
@@ -31,6 +31,15 @@ local ipairs = ipairs
 local tonumber = tonumber
 
 
+core.ctx.register_var("redis_cmd_line", function(ctx)
+    local curr_ctx = ctx.xrpc_session and ctx.xrpc_session._curr_ctx
+    if not curr_ctx then
+        core.log.warn("can't find current context")
+        return nil
+    end
+    return curr_ctx.cmd
```

Comment:
We should return cmd_line instead? https://github.com/apache/apisix/blob/1cdcc5caffc7ccf3825e3be43f62af8388b93551/apisix/stream/xrpc/protocols/redis/init.lua#L216

We can join the array with ` ` as this var is used in the log.

Reply:
> as this var is used in the log.

No, cmd_line is also used for filter, filter would be weird to write if it returned a spliced cmd_line.

cmd_line in array maybe like:

```
{ "hmget", "animals", "dog", "cat" },
```

cmd_line in string maybe like:

```
"hmget animals dog cat"
```

Reply:
Ok, I know, redis_cmd_line will only be used for logs, not filters.